websites/models.py

```diff
@@ -279,6 +279,7 @@ def get_config_file_field(self) -> Dict:
 
     def save(self, **kwargs):  # pylint: disable=arguments-differ
         """Update dirty flags on save"""
+        set_title_salutation(self)
```

Comment:
`Title` is a different field from metadata which is set differently so I'm not sure whether it's a good solution to only take salutation from metadata and append it in the title. 
In case of instructor, if title is instructor's full name, then I think, title should not be asked upon saving/updating instuctor, title should be auto-populated when instructor is saved/updated. (title = salutation + first_name + middle_name + last_name).
But since in the issue ticket, the expected behavior is: `Salutation should be part of the instructor's title when added.` so I have updated the `save` method, but I don't think this is a good approach as there might be some use-cases where this approach will cause few issues, maybe or maybe not.

Reply:
what do you think are cases where this will cause an issue?

I think it seems our assumption is that, for instructors, the `.title` field will be a sort of 'ready to go' field we can use for correctly displaying the name + title of a given instructor. If that's the intent then I think it makes sense that it's basically a computed field like `title + " " + first_name + " " + last_name`.

One suggestion to make it a little more clear what's going on would be to move the conditional you have in `set_title_salutation`: 

```python
    if (
        website_content.type == constants.CONTENT_TYPE_INSTRUCTOR
        and website_content.metadata.get("salutation", None)
    ):
```

Reply:
The exact same issues which you just pointed out below :P haha.
Another use-case is that if a course author removes the salutation from metadata then it will be still there in the title field.
Another use-case is that if a course author changes the salutation. e.g: Initially, salutation was `Dr.`, now it's changed to `Prof.` so title will become `Prof. Dr. ....`.

Hence due to these use-cases, I'm also not in the favour of this solution. 
I guess I took the `expected behavior` in the issue ticket a bit too seriously. haha